frontend/src/api/resources.ts

```diff
@@ -21,13 +21,13 @@ export const authBearerHeaderPrefix = "Bearer ";
 export const SignatureVersionV1 = "siderov1";
 export const samlSessionHeader = "saml-session";
 export const DefaultKubernetesVersion = "1.30.1";
+export const workloadProxyHostPrefix = "p";
```

Comment:
Do we need this const?

Reply:
ah, I removed the `tsgen:` hint but forgot to re-generate afterwards.